osu.Game/Overlays/Profile/Sections/BeatmapsSection.cs

```diff
@@ -22,6 +22,7 @@ public BeatmapsSection()
                 new PaginatedBeatmapContainer(BeatmapSetType.Ranked, User, UsersStrings.ShowExtraBeatmapsRankedTitle),
                 new PaginatedBeatmapContainer(BeatmapSetType.Loved, User, UsersStrings.ShowExtraBeatmapsLovedTitle),
                 new PaginatedBeatmapContainer(BeatmapSetType.Pending, User, UsersStrings.ShowExtraBeatmapsPendingTitle),
+                new PaginatedBeatmapContainer(BeatmapSetType.Guest, User, "Guest Participation Beatmaps"),
```

Comment:
Note that this is missing localisation support for the header as the resources have not been bumped yet, cc @peppy

Reply:
Want to bump resources in this PR and make sure this translation now exists?